app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -525,8 +521,8 @@ private void backToTopAppearNightly() {
   }
 
   private void initPlayStoreUri() {
-    KIWIX_LOCAL_MARKET_URI = Uri.parse("market://details?id=" + getPackageName());
-    KIWIX_BROWSER_MARKET_URI =
+    NetworkUtils.KIWIX_LOCAL_MARKET_URI = Uri.parse("market://details?id=" + getPackageName());
+    NetworkUtils.KIWIX_BROWSER_MARKET_URI =
```

Comment:
Can't we move this to the NetworkUtils also? I'm not sure whether we will get the same PackageName inside the NetworkUtils but please give it a shot. Otherwise this issue doesn't even make any sense.

Reply:
getPackageName() returns the name of application's package, the one that we specify in the manifest. So, getPackageName() will remain consistent throughout all packages.

Reply:
Thanks, @venomousboxer Please create those values inside the NetworkUtils itself.

Reply:
I'll just do it